docker/registry.go

```diff
@@ -34,10 +34,23 @@ func ListRepositoryTags(repository string) ([]string, error) {
 
 	// Make the request
 	resp, reqErr := client.Do(req)
+
 	if reqErr != nil {
 		return []string{}, errors.Wrap(reqErr, "Error requesting repositories")
 	}
 
+	// TODO Remove config suggestion and 401 check once houston is handling auth
+	if resp.StatusCode == 401 {
+
+		fmt.Println(`Failed to authenticate to registry
+	
+	You can set your registry auth using
```

Comment:
@andscoop what's up with this formatting?

Reply:
@schnie equivalent to 

`fmt.Println("Failed to authenticate to registry\n\nYou can set your registry auth using \n    astro config set docker.registry.user [registry_user] -g \n    astro config set docker.registry.password [registry_password] -g")`

I can change it, wrote it the other way to be more readable in code.